securitybot/chat/slack.py

```diff
@@ -123,4 +123,5 @@ def message_user(self, user, message):
         Sends some message to a desired user, using a User object and a string message.
         '''
         channel = self._api_call('im.open', user=user['id'])['channel']['id']
+        logging.info("Sending to '{}': '{}'".format(user['name'], message.strip().replace('\n', '\\n')))
```

Comment:
what about `repr(message.strip())` instead which would make tabs and other special chars nicely visible in the logs?